### DISCUSSION TITLE
Wanting to tweet, but source label is blank

### DISCUSSION THREAD

**Discussion by heikadog** [tweepy/tweepy#1553] [Questions] [answered]
I'm new to tweeting from Tweepy—I know the tweets post and use the right API keys, but when I post them the source label doesn't show up unlike the posts I make from other devices. My app is authorized through Twitter's developer portal and it shows up under the account I want to post it with as my app name, so I know it's working. Is this a problem which can be fixed?

**Comment by Harmon758** [comment]
Do you have an example of this?

**Comment by heikadog** [comment]
Nevermind? I guess it... fixed itself? I didn't do anything on my end, just
waited a day and the source label started showing up

Thank you!

**Comment by Harmon758** [comment, marked as answer]
This was possibly due to https://twittercommunity.com/t/resolved-issue-with-tweet-source-label-not-appearing-on-certain-tweets/152960.
